### DISCUSSION TITLE
unpackFrame_PAZ.py: Permission denied

### DISCUSSION THREAD

**Discussion by AllendePrieto** [alexisInSAR/EZ-InSAR#61] [Help]
Hi again!

When I run isce_preprocessing_SM and it calls to unpackFrame_PAZ.py script, it shows that I have no access permission (permission denied). The "README.md" file indicates that this script has been modified by EZ-InSAR. 
How can I have access permission?

Best regards!

![image](https://github.com/alexisInSAR/EZ-InSAR/assets/131383377/8cb7e915-3503-47cb-9666-cd657f03b333)

**Comment by alexisInSAR** [comment]
In your linux terminal: 

````
chmod a+x /home/callende/tools_insar/EZINSAR/EZINSAR_BIN//3rdparty/unpackFrame_PAZ.py
````

or 

````
sudo chmod a+x /home/callende/tools_insar/EZINSAR/EZINSAR_BIN//3rdparty/unpackFrame_PAZ.py
````

**Reply by AllendePrieto** [reply]
Thank you!
Now it seems to work fine, but only with VV polarization. If I choose the HH polarization, the script also looks for the VV and if it does not find it, it gives an error.
[Documento10.pdf](https://github.com/alexisInSAR/EZ-InSAR/files/12805479/Documento10.pdf)

**Reply by alexisInSAR** [reply]
Okay. I will try to fix this error tomorrow (it is linked to ISCE).
